src/SPC/builder/macos/library/glfw.php

```diff
@@ -17,11 +18,14 @@ class glfw extends MacOSLibraryBase
      */
     protected function build(): void
     {
-        // compile！
-        shell()->cd(SOURCE_PATH . '/ext-glfw/vendor/glfw')
-            ->exec("cmake . {$this->builder->makeCmakeArgs()} -DBUILD_SHARED_LIBS=OFF -DGLFW_BUILD_EXAMPLES=OFF -DGLFW_BUILD_TESTS=OFF")
-            ->exec("make -j{$this->builder->concurrency}")
-            ->exec('make install');
+        UnixCMakeExecutor::create($this)
+            ->setCMakeBuildDir("{$this->source_dir}/vendor/glfw")
```

Comment:
Rename to just `setBuildDir`. We don't need to specify cmake build dir in the cmake executor.